tokio/src/time/driver/handle.rs

```diff
@@ -47,7 +47,7 @@ cfg_rt! {
         /// panicking.
         pub(crate) fn current() -> Self {
             crate::runtime::context::time_handle()
-                .expect("there is no timer running, must be called from the context of Tokio runtime")
+                .expect("time must be enabled on the Tokio 1.x context")
```

Comment:
This error message could be improved.
```suggestion
                .expect("A Tokio 1.x context was found, but timers are disabled on the runtime.")
```